sycl/test-e2e/Matrix/element_wise_all_ops_int8_packed_impl.hpp

```diff
@@ -95,6 +100,31 @@ int main() {
           }
         }
       });
+}
+
+int main() {
+  queue q;
+  std::vector<combination> combinations =
+      q.get_device()
+          .get_info<sycl::ext::oneapi::experimental::info::device::
+                        matrix_combinations>();
+
+  for (unsigned int i = 0; i < combinations.size(); i++) {
+    if (combinations[i].nsize == 0) {                        // Intel AMX
```

Comment:
nit: why so much space?
```suggestion
    if (combinations[i].nsize == 0) { // Intel AMX
```